plugins/hydra_ax_sweeper/example/conf/config.yaml

```diff
@@ -0,0 +1,32 @@
+defaults:
+  - hydra/sweeper: ax
+
+# Banana function
+# https://en.wikipedia.org/wiki/Rosenbrock_function
+banana:
+  # To be minimized
+  x: ???
+  y: ???
+  
+
+hydra:
+  ax:
+    experiment:
+      # Banana function should be minimized (Ax defaults to maximize)
+      minimize: true
+
+    early_stop:
+      # Number of epochs without a significant improvement from
+      # the currently known best parameters
+      # An Epoch is defined as a batch of trials executed in parallel
+      max_epochs_without_improvement: 100
+    
+    params:
+      banana.x:
+        type: range
+        bounds: [-5.0, 10.0]
+        log_scale: false
+      banana.y:
+        type: choice
+        bounds: [-5.0, 5, 6]
+        log_scale: false
```

Comment:
this is the wrong type for y.
Doesn't it break the optimization?

Reply:
I was overriding via the commandline and hence did not realise. Fixed now.

Reply:
what is the bounds for 9? looks like 3 numbers.